Tools/MakePlots.C

```diff
@@ -1007,235 +1032,334 @@ int main(int argc, char* argv[])
+    Plotter::DatasetSummary dsttZ_0b("t#bar{t}Z", fileMap["TTZ"], "passBaselineNoTagZinv;passMuZinvSel;cntCSVSZinv=0", "");
+    Plotter::DatasetSummary dsVV_0b("Diboson", fileMap["Diboson"], "passBaselineNoTagZinv;passMuZinvSel;cntCSVSZinv=0", "");
+
+
+    // Define the collections, i.e. variables per selection
+    // --> met
+    Plotter::DataCollection dcData_2015CD_met_nosel("data",  "cleanMetPt", {dsData_2015C_nosel});
+    Plotter::DataCollection dcDY_met_nosel("stack",  "cleanMetPt", {dsDY_nosel, dstt2l_nosel, dstW_nosel, dsttZ_nosel, dsVV_nosel});
+    Plotter::DataCollection dcData_2015CD_met_blnotag("data",  "cleanMetPt", {dsData_2015C_blnotag});
+    Plotter::DataCollection dcDY_met_blnotag("stack",  "cleanMetPt", {dsDY_blnotag, dstt2l_blnotag, dstW_blnotag, dsttZ_blnotag, dsVV_blnotag});
+    Plotter::DataCollection dcData_2015CD_met_bl("data",  "cleanMetPt", {dsData_2015C_bl});
+    Plotter::DataCollection dcDY_met_bl("stack",  "cleanMetPt", {dsDY_bl, dstt2l_bl, dstW_bl, dsttZ_bl, dsVV_bl});
+    Plotter::DataCollection dcData_2015CD_met_0bnomt2("data",  "cleanMetPt", {dsData_2015C_0bnomt2});
+    Plotter::DataCollection dcDY_met_0bnomt2("stack",  "cleanMetPt", {dsDY_0bnomt2, dstt2l_0bnomt2, dstW_0bnomt2, dsttZ_0bnomt2, dsVV_0bnomt2});
+    Plotter::DataCollection dcData_2015CD_met_0b("data",  "cleanMetPt", {dsData_2015C_0b});
+    Plotter::DataCollection dcDY_met_0b("stack",  "cleanMetPt", {dsDY_0b, dstt2l_0b, dstW_0b, dsttZ_0b, dsVV_0b});
+    // --> ntops
+    Plotter::DataCollection dcData_2015CD_nt_nosel("data",  "nTopCandSortedCntZinv", {dsData_2015C_nosel});
+    Plotter::DataCollection dcDY_nt_nosel("stack",  "nTopCandSortedCntZinv", {dsDY_nosel, dstt2l_nosel, dstW_nosel, dsttZ_nosel, dsVV_nosel});
+    Plotter::DataCollection dcData_2015CD_nt_blnotag("data",  "nTopCandSortedCntZinv", {dsData_2015C_blnotag});
+    Plotter::DataCollection dcDY_nt_blnotag("stack",  "nTopCandSortedCntZinv", {dsDY_blnotag, dstt2l_blnotag, dstW_blnotag, dsttZ_blnotag, dsVV_blnotag});
+    Plotter::DataCollection dcData_2015CD_nt_bl("data",  "nTopCandSortedCntZinv", {dsData_2015C_bl});
+    Plotter::DataCollection dcDY_nt_bl("stack",  "nTopCandSortedCntZinv", {dsDY_bl, dstt2l_bl, dstW_bl, dsttZ_bl, dsVV_bl});
+    Plotter::DataCollection dcData_2015CD_nt_0bnomt2("data",  "nTopCandSortedCntZinv", {dsData_2015C_0bnomt2});
+    Plotter::DataCollection dcDY_nt_0bnomt2("stack",  "nTopCandSortedCntZinv", {dsDY_0bnomt2, dstt2l_0bnomt2, dstW_0bnomt2, dsttZ_0bnomt2, dsVV_0bnomt2});
+    Plotter::DataCollection dcData_2015CD_nt_0b("data",  "nTopCandSortedCntZinv", {dsData_2015C_0b});
+    Plotter::DataCollection dcDY_nt_0b("stack",  "nTopCandSortedCntZinv", {dsDY_0b, dstt2l_0b, dstW_0b, dsttZ_0b, dsVV_0b});
+    // --> ntops with bfaking
+    // Plotter::DataCollection dcData_2015CD_nt1b_0bnomt2("data", {"nTopCandSortedCntZinv1b", weight1fakebComb}, {dsData_2015C_0bnomt2});
```

Comment:
Attempt to try to use the weight while creating the DataCollection. It doesn't compile, so I'm sure I'm just missing something stupid.